test/e2e/app-dir/app-routes/app-custom-routes.test.ts

```diff
@@ -326,15 +326,19 @@ createNextDescribe(
           'should print an error when using lowercase %p in dev',
           async (method: string) => {
             await next.fetch('/lowercase/' + method)
-            expect(next.cliOutput).toContain(
-              `Detected lowercase method '${method}' in`
-            )
-            expect(next.cliOutput).toContain(
-              `Export the uppercase '${method.toUpperCase()}' method name to fix this error.`
-            )
-            expect(next.cliOutput).toMatch(
-              /Detected lowercase method '.+' in '.+\/route\.ts'\. Export the uppercase '.+' method name to fix this error\./
-            )
+
+            await check(() => {
+              expect(next.cliOutput).toContain(
+                `Detected lowercase method '${method}' in`
+              )
+              expect(next.cliOutput).toContain(
+                `Export the uppercase '${method.toUpperCase()}' method name to fix this error.`
+              )
+              expect(next.cliOutput).toMatch(
+                /Detected lowercase method '.+' in '.+\/route\.ts'\. Export the uppercase '.+' method name to fix this error\./
+              )
+              return 'yes'
```

Comment:
It’s always returning yes? The toMatch and toContain seems duplicated

Reply:
if the expect's fail it throws the return value is just to make sure it passes, we could add a different value to ensure it just doesn't throw